### PR TITLE
Introduces hostel api for maintenance staff contacts, fixes google oauth for ios devices and render flex issue.

### PR DESCRIPTION
**Pull request by architprasar**
Mess APIs Tested 
Google auth fixed for ios devices
Hostel APIs for maintenance staff contacts added

### CODE REVIEW

File: api/authentication/backends.py [hunk 1]
```diff
@@ -33,12 +33,12 @@ def authenticate(self, request):
                 return None
 
             token = authorization_header.split(' ')[1]
-
+            print(token)
```
Comment (ashutoshc8101) [inline api/authentication/backends.py:36]: Please revert this change.

File: api/hostel/urls.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                                - payload {name,contact,designation}
+                        - Delete - delete an instance using pk(primary key)
+
+"""
+
+urlpatterns = [
+    path('maintenance/contact', views.MaintenanceStaffContactsView.as_view(), name="Maintenance staff contact items"),
+    path('maintenance/contact/<int:pk>', views.MaintenanceStaffContactsInstanceView.as_view(), name="Maintenance staff contact instance items"),
+
+
```
Comment (ashutoshc8101) [inline api/hostel/urls.py:25]: Remove unnecessary empty lines.

File: api/hostel/views.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        contact = request.POST.get("contact")
+        designation = request.POST.get("designation")
+        instance = MaintenanceStaffContacts.objects.create(
+            name=name,
+            contact=contact,
+            designation=designation
+        )
+        return Response(status=200, data={
+            "msg": "Contact added successfully."
+        })
```
Comment (ashutoshc8101) [inline api/hostel/views.py:29]: Add empty line below

File: api/hostel/views.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+class MaintenanceStaffContactsInstanceView(APIView):
+    def get(self, request, *args, **kwargs):
+        contact_id = kwargs.get("pk")
+        data = MaintenanceStaffContacts.objects.filter(id=contact_id).first()
+        serialized_json = MaintenanceStaffContactsSer(data)
+        return Response(data=serialized_json.data)
+    def post(self, request, *args, **kwargs):
```
Comment (ashutoshc8101) [inline api/hostel/views.py:36]: Add empty line in between.

File: api/hostel/views.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        designation = request.POST.get("designation")
+        instance = MaintenanceStaffContacts.objects.filter(id=contact_id).update(
+            name=name,
+            contact=contact,
+            designation=designation
+        )
+        return Response(status=200, data={
+            "msg": "Contact updated successfully."
+        })
+    def delete(self, request, *args, **kwargs):
```
Comment (ashutoshc8101) [inline api/hostel/views.py:49]: ditto

File: api/mess/views.py [hunk 6]
```diff
@@ -53,7 +49,7 @@ def post(self, request, *args, **kwargs):
 class FeedbackInstanceView(APIView):
     def get(self, request, *args, **kwargs):
         feedback_id = kwargs.get("pk")
-        data = Feedback.objects.first(id=feedback_id)
+        data = Feedback.objects.first(Id=feedback_id)
```
Comment (ashutoshc8101) [inline api/mess/views.py:52]: Why `Id` is capitalized?

File: app/ios/Runner/Info.plist [hunk 7]
```diff
@@ -43,5 +43,23 @@
...
+		<key>CFBundleTypeRole</key>
+		<string>Editor</string>
+		<key>CFBundleURLSchemes</key>
+		<array>
+			<!-- TODO Replace this value: -->
+			<!-- Copied from GoogleService-Info.plist key REVERSED_CLIENT_ID -->
+			<string>com.googleusercontent.apps.907710643901-9s8q9mphvglr24hj1vogr95vdfh61c58</string>
+		</array>
+	</dict>
+</array>
```
Comment (ashutoshc8101) [inline app/ios/Runner/Info.plist:62]: Fix indentation

File: app/test_app/lib/main.dart [hunk 8]
```diff
@@ -0,0 +1,115 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
```
Comment (ashutoshc8101) [inline app/test_app/lib/main.dart:6]: Why is this file added?

File: app/test_app/.gitignore [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# Miscellaneous
+*.class
+*.log
+*.pyc
+*.swp
```
Comment (ashutoshc8101) [inline app/test_app/.gitignore:5]: I guess the test_app folder was accidentally created using `flutter create test_app`. Please delete this folder.

**Review by ashutoshc8101 — CHANGES_REQUESTED** [review]
Left few comments. @architprasar Please take a look.

**Comment by ashutoshc8101** [comment]
Update the PR title and description to concisely describe what it does/introduces.

File: api/authentication/backends.py (hunk 1 above)
Comment (architprasar) [reply api/authentication/backends.py]: done

File: api/hostel/urls.py (hunk 2 above)
Comment (architprasar) [reply api/hostel/urls.py]: done

File: api/hostel/views.py (hunk 3 above)
Comment (architprasar) [reply api/hostel/views.py:30]: done

File: api/hostel/views.py (hunk 4 above)
Comment (architprasar) [reply api/hostel/views.py]: done

File: api/hostel/views.py (hunk 5 above)
Comment (architprasar) [reply api/hostel/views.py]: done

File: api/mess/views.py (hunk 6 above)
Comment (architprasar) [reply api/mess/views.py]: done

File: app/ios/Runner/Info.plist (hunk 7 above)
Comment (architprasar) [reply app/ios/Runner/Info.plist]: done

File: app/test_app/lib/main.dart (hunk 8 above)
Comment (architprasar) [reply app/test_app/lib/main.dart]: done

File: app/test_app/.gitignore (hunk 9 above)
Comment (architprasar) [reply app/test_app/.gitignore]: deleted

**Review by iraapp — APPROVED** [review]

**Review by ashutoshc8101 — APPROVED** [review]
